sqlmesh/dbt/model.py

```diff
@@ -70,7 +69,6 @@ class ModelConfig(BaseModelConfig):
     lookback: t.Optional[int] = None
 
     # DBT configuration fields
-    start: t.Optional[str] = None
```

Comment:
why did this go away?

Reply:
It was already being inherited [here](https://github.com/TobikoData/sqlmesh/blob/main/sqlmesh/dbt/common.py#L87), so unnecessary.